src/main/java/org/apache/commons/collections4/trie/PatriciaTrie.java

```diff
@@ -29,7 +29,7 @@
  * all data at the edges of the {@link org.apache.commons.collections4.Trie}
  * (and having empty internal nodes), PATRICIA stores data in every node.
  * This allows for very efficient traversal, insert, delete, predecessor,
- * successor, prefix, range, and {@link #select(Object)}
+ * successor, prefix, range, and {@link #select(String)}
```

Comment:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:javadoc (default-cli) on project commons-collections4: An error has occurred in Javadoc report generation: 
[ERROR] Exit code: 1 - /home/travis/build/apache/commons-collections/src/main/java/org/apache/commons/collections4/trie/PatriciaTrie.java:32: error: reference not found
[ERROR]  * successor, prefix, range, and {@link #select(String)}

Maybe it's not need to update.

Reply:
@dota17 
yes you are correct.
It can pass on one of my machines with jdk8 so I thought it be correct.
but actually according to travis-ci, it can only pass openjdk8 and oraclejdk8, but fails on other jdks.
so I will revert it.
Thank you!

Reply:
@dota17
Though I have no idea why it can pass on 8 but not others....
Strange isn't it...
![image](https://user-images.githubusercontent.com/17455337/83521545-f9ab4680-a511-11ea-8ff6-9a6c1734c46b.png)